Kitodo-DataManagement/src/main/java/org/kitodo/api/ugh/UghImplementation.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.ugh;
```

Comment:
You added this a part of `org.kitodo.api.ugh` but you added this file to the `Kitodo-DataManagement` module. This is wrong. This file belongs to the `Kitodo-API`module.

Reply:
This is the entry point to the ugh. Adding it to the API module would mean that the Kitodo-API has a dependency to the UGH. This is probably not what we want. However, the package name needs to change. I just don’t have a good idea how to call it.

Reply:
So if it is necessary that someone depends on UGH it should be API not Data Management or Core. It is a whole point of interfaces and modules... Why should we even use interface classes if still we can fully access the whole UGH?